selftests/pre_release/jobs/pre_release.py

```diff
@@ -28,6 +28,8 @@
 
 if __name__ == '__main__':
     os.chdir(ROOT_DIR)
-    with Job.from_config({}, [cirrus_ci, parallel_1, vmimage]) as j:
+    config = {'job.output.testlogs.statuses': ['FAIL', 'ERROR', 'INTERRUPT'],
+              'job.output.testlogs.logfiles': ['debug.log']}
```

Comment:
Currently `job.output.testlogs.logfiles`  has as default `['debug.log']`. So, I'm just curious if we really need this second option.

Reply:
yes, you are right, I just copied it from `selftest/check.py` so I think we can remove it there too. Thanks